libfreerdp/core/tcp.c

```diff
@@ -145,6 +145,8 @@ static int transport_bio_simple_read(BIO* bio, char* buf, int size)
 
 	BIO_clear_flags(bio, BIO_FLAGS_READ);
 
+	WSAResetEvent(ptr->hEvent);
```

Comment:
Same as listener.c
We probably need to keep my '#ifdef _WIN32' here to only reset on windows platform